sci-biology/sailfish/files/sailfish-0.9.1-no-boost-static.patch

```diff
@@ -0,0 +1,20 @@
+Do not force static and multithreaded boost libs, use what is available
+
+Patch by Michael Schubert
+
+--- CMakeLists.txt
++++ CMakeLists.txt
```

Comment:
It would be a good idea for future maintainability if those patch were `git` style
```
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
```
It doesn't have to be `a` and `b` but in the future with EAPI 6 and the `eapply` helper which is replacing `epatch`, `patch` will only be called with `-p1`, i.e. with a leading directory being stripped. The current `epatch` function tries everything between 0 and 10 (I think).